acceptance/suites/tests/service_framework/error_during_restart.rb

```diff
@@ -1,33 +0,0 @@
-test_name 'Service returns error if restarted and puppetserver errors out'
-
-puppetservice=options['puppetservice']
-bogus_config_file='/etc/puppetlabs/puppetserver/conf.d/bogus.conf'
-
-teardown do
-  step 'Remove bogus config file' do
-    on(master, "rm -f #{bogus_config_file}")
-  end
-
-  step 'Restart puppet server for next test' do
-    bounce_service(master, puppetservice)
-  end
-end
-
-step 'Create config file with bogus data' do
-  on(master, "echo '12345' > #{bogus_config_file}")
-end
-
-step 'Ensure puppetserver fails to start' do
-  on(master, "service #{puppetservice} stop")
-  on(master, "service #{puppetservice} start", {:acceptable_exit_codes => [1]})
-end
-
-step 'Ensure puppetserver fails to start after a restart' do
-  # Sleep a few seconds to try to provoke contention between the automatic
-  # restart that systemd would do - for restart=on-failure - and the
-  # 'manually requested' one below.  The 'manually requested' restart should
-  # complete but with an error (1) status code.
```

Comment:
From hipchat room: This is really testing trapperkeeper's restart behavior, not so much server behavior. We should rely on trapperkeeper's tests for this.